rfcs/IncrementalDelivery.md

```diff
@@ -0,0 +1,45 @@
+# Incremental Delivery over HTTP
+
+This RFC proposes adding a specification of how "Incremental" GraphQL results should be delivered to clients, using HTTP chunked transfer encoding.
+
+## Incremental Results
+
+An Incremental result is a GraphQL result that is split up into multiple payloads, allowing clients quicker access to parts of the results. Currently this is supported by the proposed `@defer` and `@stream` directives ([RFC](https://github.com/graphql/graphql-spec/blob/master/rfcs/DeferStream.md)).
+
+## `transfer-encoding: chunked`
+
+The HTTP response for an incrementally delivered response should contain the `transfer-encoding: chunked` response header. Chunked transfer encoding allows the body of the response to be delivered as a series of chunks, allowing clients to read each chunk of the response as it is sent without waiting for the entire response.
+
+## `content-type: multipart/mixed`
+
+The HTTP response for an incrementally delivered response should contain the `content-type: multipart/mixed; boundary="-"` response header and conform to the [specification of multipart content defined by the W3 in rfc1341](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html). An example response body will look like:
+
+```
+---
+Content-Type: application/json; charset=utf-8
+Content-Length: 45
+
+{"data":{"hello":"Hello Rob"},"hasNext":true}
+
+---
+Content-Type: application/json; charset=utf-8
+Content-Length: 57
+
+{"data":{"test":"Hello World"},"path":[],"hasNext":false}
+
+-----
+```
+* The boundary used is `-` and is passed to the client in the http response's content-type header. 
```

Comment:
I co not see this in the example above.

Reply:
The `content-type: multipart/mixed; boundary="-"` header should be in the HTTP response headers. The example is the HTTP response body. Each part of the response body contains it's own set of headers, but the header defining the boundary needs to be on the HTTP response, not in the body.

Reply:
it's worth mentioning